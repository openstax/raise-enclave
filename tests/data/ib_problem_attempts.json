[
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "http://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_pset_problem_attempted_v1",
        "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
        "variant": "main",
        "pset_content_id": "5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12",
        "pset_problem_content_id": "743cb23f-527b-44b8-ac3b-39f88f8885b0",
        "problem_type": "input",
        "response": "some user input string",
        "correct": false,
        "attempt": 1,
        "final_attempt": false
    },
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "http://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_pset_problem_attempted_v1",
        "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
        "variant": "main",
        "pset_content_id": "5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12",
        "pset_problem_content_id": "743cb23f-527b-44b8-ac3b-39f88f8885b0",
        "problem_type": "dropdown",
        "response": "selected input",
        "correct": false,
        "attempt": 1,
        "final_attempt": false
    },
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "http://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_pset_problem_attempted_v1",
        "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
        "variant": "main",
        "pset_content_id": "5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12",
        "pset_problem_content_id": "743cb23f-527b-44b8-ac3b-39f88f8885b0",
        "problem_type": "multiselect",
        "response": ["selected input 1", "selected input 2"],
        "correct": false,
        "attempt": 1,
        "final_attempt": false
    },
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "http://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_pset_problem_attempted_v1",
        "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
        "variant": "main",
        "pset_content_id": "5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12",
        "pset_problem_content_id": "743cb23f-527b-44b8-ac3b-39f88f8885b0",
        "problem_type": "multiplechoice",
        "response": "selected input",
        "correct": false,
        "attempt": 1,
        "final_attempt": false
    }
]