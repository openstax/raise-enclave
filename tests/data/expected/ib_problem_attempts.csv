user_uuid,course_id,impression_id,timestamp,content_id,pset_content_id,pset_problem_content_id,variant,problem_type,response,correct,attempt,final_attempt
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12,743cb23f-527b-44b8-ac3b-39f88f8885b0,main,input,some user input string,False,1,False
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12,743cb23f-527b-44b8-ac3b-39f88f8885b0,main,dropdown,selected input,False,1,False
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12,743cb23f-527b-44b8-ac3b-39f88f8885b0,main,multiselect,"['selected input 1', 'selected input 2']",False,1,False
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,3fa85f64-5717-4562-b3fc-2c963f66afa7,5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12,743cb23f-527b-44b8-ac3b-39f88f8885b0,main,multiplechoice,selected input,False,1,False